internal/common/common.go

```diff
@@ -609,3 +609,7 @@ func IsExternalIntegrationEnabled(platform *models.Platform, platformName string
 func IsOciExternalIntegrationEnabled(platform *models.Platform) bool {
 	return IsExternalIntegrationEnabled(platform, ExternalPlatformNameOci)
 }
+
+func IsNonePlatformCluster(cluster *Cluster) bool {
+	return !IsSingleNodeCluster(cluster) && swag.BoolValue(cluster.UserManagedNetworking)
```

Comment:
Why exclude SNO? while it's not relevant to the fix in this PR SNO is a none platform cluster.
Consider renaming the function to `IsMultiNodeNonePlatformCluster`

Reply:
OK